components/nodemanager-service/pgdb/node-managers-process-node.go

```diff
@@ -127,20 +127,12 @@ func (db *DB) ProcessIncomingNode(node *manager.NodeMetadata) error {
 		return errors.Wrap(err, "ProcessIncomingNode unable to marshal projects data")
 	}
 
-	var mgrType string
-	if node.ManagerId != "" {
-		mgrType, err = db.SelectStr(sqlGetManagerTypeFromId, node.ManagerId)
-		if err != nil {
-			logrus.Warnf("unable to find manager for node")
```

Comment:
Should we still warn here if `node.GetManagerType()` is empty?

Reply:
hmm, good call ya